test/component/testSignOut.js

```diff
@@ -0,0 +1,26 @@
+const TestWrapper = require('test/util/TestWrapper');
+const config = require('app/config');
+
+describe('sign-out', () => {
+    let testWrapper;
+
+    beforeEach(() => {
+        testWrapper = new TestWrapper('SignOut');
+    });
+
+    afterEach(() => {
+        testWrapper.destroy();
+    });
+
+    describe('Verify Content, Errors and Redirection', () => {
+
+        it('test right content loaded on the page', (done) => {
+            const excludeKeys = [];
+            const contentData = {
+                idamLoginLink: config.services.idam.loginUrl
+            };
```

Comment:
Do we need to define this `contentData` as there doesn't appear to be any content that needs to be replaced?